<launch>
  <arg name="serialport"  default="/dev/aimibot" />
  <param name="vel_line_"  value="0.1"/>
  <param name="vel_angle_" value="0.5"/>

  <node pkg="aimibot" type="aimibot" name="aimibot" output="screen">
    <param name="serial_port" value="$(arg serialport)"/>
    <param name="odom_frame"      value="odom"/>
    <param name="base_frame"      value="base_footprint"/>
    <param name="use_imu_heading" value="true" />
    <param name="publish_tf"      value="true" />
    <remap from="/mobile_base/commands/velocity" to="/aimibot/commands/velocity"/>
  </node>
  
  <node pkg="aimibot" type="movecircular" name="movecircular"  output="screen">
    <remap from="/mobile_base/commands/velocity" to="/aimibot/commands/velocity"/>
  </node>
</launch>
